src/main/java/net/openhft/chronicle/bytes/MappedBytesStore.java

```diff
@@ -479,7 +479,7 @@ public void syncMode(SyncMode syncMode) {
      * @param position to sync with the syncMode()
      */
     public void syncUpTo(long position) {
-        if (syncMode == SyncMode.NONE || address == 0 || refCount() <= 0)
+        if (syncMode == SyncMode.NONE || address == 0 || refCount() <= 0 || !OS.isLinux())
```

Comment:
Hi @peter-lawrey, you may be able to replace with `isUnix` or alternatively `!OS.isWindows()` (don't believe there is an `isUnix` in core. In other words this can likely work on Linux, Mac, other Unix like OS supporting `msync`.

Here's the code in the `Posix` API that switches the underlying Posix wrapper depending on platform:

https://github.com/OpenHFT/Posix/blob/ea/src/main/java/net/openhft/posix/internal/PosixAPIHolder.java#L17-L19

This ultimately resolves down to:

```java
    public final boolean isUnix() {
        return this.os != Platform.OS.WINDOWS;
    }
```